src/lv_conf_internal.h

```diff
@@ -29,7 +29,7 @@
 
 /*If lv_conf.h is not skipped include it*/
 #if !defined(LV_CONF_SKIP)
-#  if defined(LV_CONF_PATH)											/*If there is a path defined for lv_conf.h use it*/
+#  if defined(LV_CONF_PATH)                                         /*If there is a path defined for lv_conf.h use it*/
```

Comment:
src/lv_conf_internal.h is automatically generated too. Please modify https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_checker.py instead.